web/html/src/branding/css/susemanager/components/buttons.scss

```diff
@@ -74,3 +74,7 @@ button.is-plain {
     }
   }
 }
+
+.input-group-btn .btn {
+  padding: 10px 9px;
```

Comment:
This does not look like it can be right. The paddings used in the Suma theme and Uyuni theme are different, I doubt this value works for both?

Reply:
I will have a look